### ISSUE TITLE
Pagination Accessibility

### ISSUE THREAD

**Issue by dianaOSI** [ca-cwds/design-system#52]
Automated assessment using AXE and WAVE reveals some critical accessibility issues with the pagination controls. These are the priority 1 critical issues to resolve.
Pagination considerations: http://www.a11ymatters.com/pattern/pagination/

### Issues:
**1. Certain ARIA roles must contain particular children**
<img width="491" alt="screen shot 2018-09-11 at 2 53 42 pm" src="https://user-images.githubusercontent.com/22502707/45390099-d2d90e00-b5d2-11e8-9594-40e36976e9b2.png">
How to Fix: https://dequeuniversity.com/rules/axe/3.0/aria-required-children?application=AxeChrome

**2. Certain ARIA roles must be contained by particular parents**
<img width="387" alt="screen shot 2018-09-11 at 3 00 26 pm" src="https://user-images.githubusercontent.com/22502707/45390266-71fe0580-b5d3-11e8-8e66-493f1fcd88ff.png">
How to Fix: https://dequeuniversity.com/rules/axe/3.0/aria-required-parent?application=AxeChrome

**3. ARIA attributes must conform to valid values**
<img width="390" alt="screen shot 2018-09-11 at 3 02 00 pm" src="https://user-images.githubusercontent.com/22502707/45390342-b1c4ed00-b5d3-11e8-9233-2a38c60ce471.png">
How to Fix:  https://dequeuniversity.com/rules/axe/3.0/aria-valid-attr-value?application=AxeChrome

**4. Buttons must have discernible text**
<img width="387" alt="screen shot 2018-09-11 at 3 03 22 pm" src="https://user-images.githubusercontent.com/22502707/45390385-f18bd480-b5d3-11e8-924d-da74a07a9913.png">
How to Fix: https://dequeuniversity.com/rules/axe/3.0/button-name?application=AxeChrome

**5. Form elements must have labels**
<img width="379" alt="screen shot 2018-09-11 at 3 05 42 pm" src="https://user-images.githubusercontent.com/22502707/45390457-34e64300-b5d4-11e8-826d-0f7415ac8740.png">
How to Fix: https://dequeuniversity.com/rules/axe/3.0/label?application=AxeChrome

Recheck using AXE and filter results by "Violations" to see if they are resolved.

**Comment by eckdanny-osi** [comment]
Oops. Automatically closed by PR message body, which only addresses points 4 and 5 in the issue description.

> 4. Buttons must have discernible text
> 5. Form elements must have labels

Reopening.

**Comment by denys-davydov** [comment]
I will add Capybara scan result for CANS project. 3 issues here: 

`Found 3 accessibility violations:

       1) aria-required-children: Certain ARIA roles must contain particular children (critical)
           https://dequeuniversity.com/rules/axe/3.0/aria-required-children?application=axeAPI
           The following 2 nodes violate this rule:

               Selector: #react-select-2--value > .Select-input[aria-owns=""][role="combobox"]
               HTML: <div aria-expanded="false" aria-owns="" aria-activedescendant="react-select-2--value" aria-disabled="false" class="Select-input" role="combobox" tabindex="0" style="border: 0px; width: 1px; display: inline-block;"></div>
               Fix any of the following:
               - Required ARIA child role not present: textbox

               Selector: #react-select-3--value > .Select-input[aria-owns=""][role="combobox"]
               HTML: <div aria-expanded="false" aria-owns="" aria-activedescendant="react-select-3--value" aria-disabled="false" class="Select-input" role="combobox" tabindex="0" style="border: 0px; width: 1px; display: inline-block;"></div>
               Fix any of the following:
               - Required ARIA child role not present: textbox

       2) aria-required-parent: Certain ARIA roles must be contained by particular parents (critical)
           https://dequeuniversity.com/rules/axe/3.0/aria-required-parent?application=axeAPI
           The following 2 nodes violate this rule:

               Selector: #react-select-2--value-item
               HTML: <span class="Select-value-label" role="option" aria-selected="true" id="react-select-2--value-item">10 rows</span>
               Fix any of the following:
               - Required ARIA parent role not present: listbox

               Selector: #react-select-3--value-item
               HTML: <span class="Select-value-label" role="option" aria-selected="true" id="react-select-3--value-item">10 rows</span>
               Fix any of the following:
               - Required ARIA parent role not present: listbox

       3) aria-valid-attr-value: ARIA attributes must conform to valid values (critical)
           https://dequeuniversity.com/rules/axe/3.0/aria-valid-attr-value?application=axeAPI
           The following 2 nodes violate this rule:

               Selector: #react-select-2--value > .Select-input[aria-owns=""][role="combobox"]
               HTML: <div aria-expanded="false" aria-owns="" aria-activedescendant="react-select-2--value" aria-disabled="false" class="Select-input" role="combobox" tabindex="0" style="border: 0px; width: 1px; display: inline-block;"></div>
               Fix all of the following:
               - Invalid ARIA attribute value: aria-owns=""

               Selector: #react-select-3--value > .Select-input[aria-owns=""][role="combobox"]
               HTML: <div aria-expanded="false" aria-owns="" aria-activedescendant="react-select-3--value" aria-disabled="false" class="Select-input" role="combobox" tabindex="0" style="border: 0px; width: 1px; display: inline-block;"></div>
               Fix all of the following:
               - Invalid ARIA attribute value: aria-owns=""`

**Comment by eckdanny-osi** [comment]
The 3 a11y bugs from @denys-davydov seem to be consistent with @dianaOSI's reporting.

I'm guessing this may be an implementation detail in `react-select`. We should check the `react-select` issue log and hope its been resolved and updated version of the `1.x` lib is available. 

Note: `react-select`@`2.x` stable has been available for over 2 months now. I'm not sure what the support outlook is for 1.x but we should probably focus efforts on updating.

**Comment by eckdanny-osi** [comment]
related to #159
